dynamic/util/.gitignore

```diff
@@ -0,0 +1 @@
+crashReport.sh
```

Comment:
create the `util` directory, so that when a customer would use these scripts and Dockerfiles to create their own containers but doesn't realize the `crashReport` script needs to be copied, it will still work.